index.html

```diff
@@ -0,0 +1,175 @@
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+        <div class="grid-item">
+            <div class="grid-item-photo" id="photo4">
+            </div>
+            <section class="grid-item-section">
+                <h2 class="title">Multi-Post Stories</h2>
+                <ul class="sub-title">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>
+                    A daily selection of privately personalized reads; no account or sign-ups required.</p>
+                <ul class="category">
+                    <li class="category-list"><a href="index.html">html</a></li>
+                    <li class="category-list"><a href="index.html">css</a></li>
+                    <li class="category-list"><a href="index.html">javaScript</a></li>
+                </ul>
+                <button type="submit" class="project">See Project</button>
+            </section>
+        </div>
+    </div>
+    <section class="about" id="about">
+        <div class="about-content1">
+            <h2>About Myself</h2>
+            <p>
+                Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <ul class="social-1">
+                <li><a href="index.html"><img src="image/1.png" alt="Twitter"/></a></li>
+                <li><a href="index.html"><img src="image/2.png" alt="Linkedin"/></a></li>
+                <li><a href="index.html"><img src="image/3.png" alt="Medium"/></a></li>
+                <li><a href="index.html"><img src="image/4.png" alt="Github"/></a></li>
+                <li><a href="index.html"><img src="image/5.png" alt="Angellist"/></a></li>
+            </ul>
+            <button type="submit" class="project">Get my resume</button>
+        </div>
+        <div class="about-content2">
+            <ul class="skills">
+                <li id="list"><a href="index.html">Languages</a></li>
+                <div class="extra">
+                        <a href="index.html">
+                            <img src="image/Ellipse-1.png" alt="JavaScript" />
+                            <p class="list-4">JavaScript</p>
+                        </a>
+                        <a href="index.html">
+                            <img src="image/Ellipse-3.png" alt="HTML" />
+                            <p>HTML</p>
+                        </a>
+                        <a href="index.html">
+                            <img src="image/Ellipse-2.png" alt="Css" />
+                            <p>css</p>
+                        </a>
+                    </li>
+                </div>
```

Comment:
- [ ] Kindly adjust the paths of your icons here in your about me section to display the icons 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/213791281-6d4b9b01-3e93-44ae-ac8b-ff0e26c2bdb8.png)